tfx/orchestration/pipeline.py

```diff
@@ -189,3 +190,76 @@ def components(self, components: List[base_node.BaseNode]):
     # has all its dependencies visited.
     if len(self._components) < len(deduped_components):
       raise RuntimeError('There is a cycle in the pipeline')
+
+  def get_fuseable_components(self):
```

Comment:
nit: there's a typo in the PR name; please change it to match this method name.

Reply:
Changed the PR name to reflect the new class `BeamFusionOptimizer()`.